common/src/main/java/com/viaversion/viaversion/protocols/protocol1_9to1_8/packets/PlayerPackets.java

```diff
@@ -300,22 +307,6 @@ public void registerMap() {
             }
         });
 
-        protocol.registerClientbound(ClientboundPackets1_8.UPDATE_HEALTH, new PacketRemapper() {
```

Comment:
Is this not/no longer necessary?

Reply:
The client doesn't unload chunks when it gets a health = 0 packet, it only shows the death screen, the respawn packet unload the chunks when the client presses respawn